core/io/http_client.cpp

```diff
@@ -63,8 +63,9 @@ Error HTTPClient::_request_raw(Method p_method, const String &p_url, const Vecto
 }
 
 Error HTTPClient::_request(Method p_method, const String &p_url, const Vector<String> &p_headers, const String &p_body) {
-	int size = p_body.length();
-	return request(p_method, p_url, p_headers, size > 0 ? (const uint8_t *)p_body.utf8().get_data() : nullptr, size);
+	CharString cs = p_body.utf8();
```

Comment:
```suggestion
	CharString body_utf8 = p_body.utf8();
```
For more descriptive names

Reply:
I looked up CharString and saw most people just call it 'cs'. So I figured I'd copy that too. Since it only shows up in the two lines below, such a short name should still be fine here as well imo. I'll wait for further review so checks don't rerun just for this.

Reply:
It's an old habit, but generally names should be indeed descriptive. It's not a huge deal in such a small method, but if you could update it, it would be great.

Reply:
ok, updated it. Thanks for the better name suggestion Redwork.